test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -119,69 +119,79 @@ if (dialect.startsWith('postgres')) {
         {
           arguments: ['myTable', { int: 'INTEGER', bigint: 'BIGINT', smallint: 'SMALLINT' }],
           expectation: 'CREATE TABLE IF NOT EXISTS "myTable" ("int" INTEGER, "bigint" BIGINT, "smallint" SMALLINT);',
+          context: { options: { databaseVersion: '11.0.0' } },
```

Comment:
This should not be necessary anymore with https://github.com/sequelize/sequelize/pull/15249/files#diff-d6c44e5861b8b76b4562220dbed9b2043de963db1178b62507aaee6bc0dfdfdb
But maybe we should set the databaseVersion for tests to the minimal supported version by default?

Reply:
Yeah this makes sense! Lemme see and make that change, since for unit tests we can't really fetch the version of the db. Also the reason I had to do this was Postgres, it threw `null` is not a valid version!